keyboards/mmkeyboard/class60/soldered/info.json

```diff
@@ -0,0 +1,960 @@
+{
+    "manufacturer": "MMkeyboard",
+    "keyboard_name": "Class60",
+    "maintainer": "lexbrugman",
+    "bootloader": "uf2boot",
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": true,
+        "bootmagic": true,
+        "extrakey": true,
+        "haptic": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "B9",
+        "num_lock": "B8",
+        "on_state": 0,
+        "scroll_lock": "B11"
+    },
+    "matrix_pins": {
+        "cols": ["B1", "B0", "A7", "A1", "A2", "A3", "A4", "A5", "B12", "B13", "A10", "A9", "A8", "B14", "B15"],
+        "rows": ["B6", "B7", "C13", "B10", "C14", "B5"]
+    },
+    "processor": "STM32F103",
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
```

Comment:
```suggestion
    },
    "audio": {
        "voices": false
    },
```

Reply:
Ended up dropping this as the default is already `false` according to the documentation